tests/utils/test_databricks_utils.py

```diff
@@ -535,3 +536,11 @@ def test_get_workspace_url(input_url, expected_result):
     with mock.patch("mlflow.utils.databricks_utils._get_workspace_url", return_value=input_url):
         result = get_workspace_url()
         assert result == expected_result
+
+
+def test_get_databricks_creds_config_ignore_error():
+    config = _get_databricks_creds_config("databricks", ignore_error=True)
+    assert isinstance(config, DatabricksConfig)
+
+    with pytest.raises(MlflowException, match="Reading databricks credential configuration failed"):
+        _get_databricks_creds_config("databricks")
```

Comment:
can we swap the order of the assertions to test the success case before the failure case? This will help ensure that the valid configuration is checked first. Here's the updated code:

```python
config = _get_databricks_creds_config("databricks", ignore_error=True)
assert isinstance(config, DatabricksConfig)

with pytest.raises(MlflowException, match="Reading databricks credential configuration failed"):
    _get_databricks_creds_config("databricks")
```